### PR TITLE
Remove sign in and sign up

### PR DESCRIPTION
**Pull request by vpetrusenko**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -12,7 +12,6 @@ gem 'rotp'
 gem 'json'
 gem 'jbuilder'
 
-gem 'omniauth',               '~> 1.2.1'
```
Comment (yivo) [inline Gemfile]: @vpetrusenko You should keep this gem. It is necessary for OAuth to work. We don't need `omniauth-identity` anymore.

File: app/controllers/identities_controller.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@ class IdentitiesController < ApplicationController
   before_filter :auth_anybody!, only: :new
```
Comment (yivo) [inline app/controllers/identities_controller.rb]: @vpetrusenko We probably don't need this controller at all.

File: app/views/welcome/index.html.erb [hunk 3]
```diff
@@ -18,10 +18,6 @@
       <% offset -= 2 if ENV['GOOGLE_OAUTH2_SIGN_IN'] %>
```
Comment (yivo) [inline app/views/welcome/index.html.erb]: @vpetrusenko I will help you with styling these buttons. We need only to update offsets.

File: config/initializers/omniauth.rb [hunk 4]
```diff
@@ -16,23 +16,3 @@
...
   end
 end
-
-OmniAuth.config.on_failure = lambda do |env|
-  SessionsController.action(:failure).call(env)
-end
-
-OmniAuth.config.logger = Rails.logger
-
-module OmniAuth
```
Comment (yivo) [inline config/initializers/omniauth.rb]: @vpetrusenko This is OK. No more `identity` strategy should exist.

File: config/initializers/omniauth.rb [hunk 5]
```diff
@@ -16,23 +16,3 @@
     provider :google_oauth2, ENV.fetch('GOOGLE_CLIENT_ID'), ENV.fetch('GOOGLE_CLIENT_SECRET')
   end
 end
-
-OmniAuth.config.on_failure = lambda do |env|
```
Comment (yivo) [inline config/initializers/omniauth.rb]: @vpetrusenko This should be kept.

File: config/initializers/omniauth.rb [hunk 6]
```diff
@@ -16,23 +16,3 @@
...
-  SessionsController.action(:failure).call(env)
-end
-
-OmniAuth.config.logger = Rails.logger
```
Comment (yivo) [inline config/initializers/omniauth.rb]: @vpetrusenko This should be kept.

File: db/migrate/20130810162023_create_reset_passwords.rb [hunk 7]
```diff
@@ -1,23 +0,0 @@
-class CreateResetPasswords < ActiveRecord::Migration
```
Comment (yivo) [inline db/migrate/20130810162023_create_reset_passwords.rb]: @vpetrusenko Stop! Migrations are READONLY. Don't delete them!.

**Review by yivo — CHANGES_REQUESTED** [review]
- [ ] no migrations for password column, tokens, password resets and lot more stuff. Please install connect to mysql server and check what schema we have.
- [ ] Auth using OAuth protocol should be kept.
- [ ] Tests for signin & signup should be removed, existing tests which use signin should be updated.
- [ ] lot of views related to signin/up are still in the project
Please, use IDE.

File: config/routes.rb [hunk 8]
```diff
@@ -10,22 +10,17 @@ def draw(routes_name)
...
-  get '/signin' => 'sessions#new', :as => :signin
-  get '/signup' => 'identities#new', :as => :signup
   get '/signout' => 'sessions#destroy', :as => :signout
   get '/auth/failure' => 'sessions#failure', :as => :failure
   match '/auth/:provider/callback' => 'sessions#create', via: [:get, :post]
 
   resource :member, :only => [:edit, :update]
-  resource :identity, :only => [:edit, :update]
 
   namespace :authentications do
```
Comment (yivo) [inline config/routes.rb]: @vpetrusenko We can remove this set of routes completely and also remove `AuthenticationsController`.

File: config/routes.rb [hunk 9]
```diff
@@ -10,22 +10,17 @@ def draw(routes_name)
...
     resources :emails, only: [:new, :create]
-    resources :identities, only: [:new, :create]
   end
 
   scope :constraints => { id: /[a-zA-Z0-9]{32}/ } do
```
Comment (yivo) [inline config/routes.rb]: @vpetrusenko This set of routes can also be removed.

**Review by yivo — CHANGES_REQUESTED** [review]

File: config/routes.rb (hunk 9 above)
Comment (vpetrusenko) [reply config/routes.rb]: @yivo it's done

File: Gemfile [hunk 10]
```diff
@@ -12,7 +12,6 @@ gem 'rotp'
 gem 'json'
 gem 'jbuilder'
 
-gem 'omniauth',               '~> 1.8.1'
```
Comment (yivo) [inline Gemfile]: @vpetrusenko please bring back this gem!

File: Gemfile [hunk 11]
```diff
@@ -12,7 +12,6 @@ gem 'rotp'
...
 gem 'omniauth-identity',      '~> 1.1.1'
```
Comment (yivo) [inline Gemfile]: @vpetrusenko remove this gem

File: config/routes.rb [hunk 12]
```diff
@@ -10,22 +10,11 @@ def draw(routes_name)
...
 
-  resource :member, :only => [:edit, :update]
-  resource :identity, :only => [:edit, :update]
-
-  namespace :authentications do
-    resources :emails, only: [:new, :create]
-    resources :identities, only: [:new, :create]
-  end
-
   scope :constraints => { id: /[a-zA-Z0-9]{32}/ } do
```
Comment (yivo) [inline config/routes.rb]: @vpetrusenko fully remove this

**Review by yivo — CHANGES_REQUESTED** [review]

**Review by yivo — APPROVED** [review]

**Pull request merged**